multiwallet.py

```diff
@@ -469,6 +493,60 @@ def do_generate_seed(self, arg):
             ),
         )
 
+    def do_blind_xpub(self, arg):
+        """Blind an XPUB using a random BIP32 path. Experts only!"""
+
+        # Prevent footgun-ing
+        if not self.ADVANCED_MODE:
+            warning_msg = (
+                "Blinding an xpub must be performed correctly or you could lose funds!\n"
+                "Multiwallet supports the custom BIP32 paths needed, but few HWWs can sign on these paths.\n"
+                "Some can't even co-sign a multisig transaction with nonstandard BIP32 paths (even if their path is standard)"
+            )
+            _print_footgun_warning(warning_msg)
+            return
+
+        key_record_dict = _get_key_record()
+        xfp_hex = key_record_dict["xfp"]
+        starting_xpub = key_record_dict["xpub"]
+        starting_path = key_record_dict["path"]
+
+        # Get secret BIP32 path:
+        depth = _get_int(
```

Comment:
I would leave off this feature, personally. Asking users to define parameters when they don't understand the tradeoffs is almost always bad UI.

Reply:
Hmmm, I would normally bury this input in the advanced section, but this whole feature is buried in the advanced section. I just created some new UI logic that I think addresses this concern nicely:
https://github.com/buidl-bitcoin/buidl-python/pull/52/commits/4a491a15d0b3e55b90b3a1d84f59b282767b7808